ckanext/dcat/tests/test_euro_dcatap_profile_serialize.py

```diff
@@ -606,13 +610,17 @@ def test_distribution_fields(self):
         # List
         for item in [
             ('documentation', FOAF.page, URIRef),
-            ('language', DCT.language, Literal),
+            ('language', DCT.language, [Literal, Literal, URIRef]),
```

Comment:
This should probably be `[Literal, URIRef]`

Reply:
`Literal` is contained twice because a type is assigned to each item in the test value list. As the list (line 575, the one above in the diff) contains three items, the data type list contains three items as well.

Reply:
Okay got it. Thanks for explaining.

Reply:
You're welcome!